docker/compose/dev-sampledata.yml

```diff
@@ -6,7 +6,7 @@ services:
       context: ../../
       dockerfile: docker/compose/mozdef_sampledata/Dockerfile
     restart: always
-    command: bash -c '/opt/mozdef/envs/mozdef/examples/demo/sampleevents.sh'
+    command: bash -c 'source /opt/mozdef/envs/python/bin/activate && /opt/mozdef/envs/mozdef/examples/demo/sampleevents.sh'
```

Comment:
The source command is no longer needed as I've added the virtualenvironment path to the base container.